src/main/java/seedu/notus/data/notebook/Notebook.java

```diff
@@ -226,7 +226,7 @@ public Note getNote(int index) {
      * @return note that was filtered.
      */
     public Note getNote(String noteTitle, boolean isArchive) {
-        if (!isArchive) {
+        if (isArchive) {
```

Comment:
Is this function called in other methods? Changing this boolean may affect the output of others. Also, from the parameter name, it sound logical that if it is not isArchive, it will search through the notes arrayList as described in the JavaDoc comment.

Reply:
Suggested change should be inside the archiveNotes(String noteTitle) under notebook.

Note archivedNote = getNote(noteTitle, true);  ----> Note toBeArchiveNote= getNote(noteTitle, false);